server/public/pluginapi/store.go

```diff
@@ -62,20 +63,22 @@ func (s *StoreService) Close() error {
 	s.mutex.Lock()
 	defer s.mutex.Unlock()
 
-	if !s.initialized {
+	// replicaDB will be nil if it's not initialized, so no need
+	// to check for initialization again.
```

Comment:
Absolutely, yes: mostly just wondering if it would make sense to check `s.masterDB != nil` too for simplicity instead of inferring it from `initializedMaster`. -1/5.

Reply:
Ah yes, gotcha